src/__tests__/__snapshots__/keyring.test.js.snap

```diff
@@ -51,11 +59,19 @@ Object {
 exports[`Keyring Keyring from seed derives correct keys from 2`] = `
 Object {
   "asymEncryptionKey": "jRKRy3oPhXpUBrFJefO2CJnWw/IoalOPuepAwvTqrEk=",
-  "managementKey": "0xa7b76379659b6c19E4DFf46e0DfEa48AFEBA8fe3",
+  "managementKey": "0350966498e802a007513d518dc12a0a0ab1e48addcec1beaa989c6ff5f5ed226c",
```

Comment:
This key value format is different from the other added snapshots, is this expected?

Reply:
Well spotted! Yeah, looks like the snapshot 2 here is swapped with snapshot 3. And previously the `uncompressed` param was passed incorrectly to the method that was being tested (thus only the `managementKey` being wrong here)